gcloud-java-logging/baseline/src/main/java/com/google/gcloud/logging/spi/v2/LoggingServiceV2Api.java

```diff
@@ -311,24 +306,40 @@ public WriteLogEntriesResponse writeLogEntries(WriteLogEntriesRequest request) {
    *
    * <!-- manual edit -->
    * <!-- end manual edit -->
+   * @throws ApiException if the remote call fails
```

Comment:
Forgot to mention that in the earlier PR.

This method will **never** throw, right?

I think you should fix the ApiCallable javadoc to make it clear that its synchronous method, `call(...)` can throw `ApiException` and that the asynchronous ones, `futureCall` returns a future that can fail with `ApiException`.